README.md

```diff
@@ -27,12 +27,35 @@ list(APPEND CMAKE_PREFIX_PATH ${CMAKE_BINARY_DIR})
 
 add_definitions("-std=c++11")
 
+set (CONAN_CMAKE_VERSION "0.17.0")
+set (CONAN_CMAKE_HASH 3bef79da16c2e031dc429e1dac87a08b9226418b300ce004cc125a82687baeef)
+
+set (DOWNLOAD_CONAN_CMAKE false)
+
 if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
+  set (DOWNLOAD_CONAN_CMAKE true)
+else()
+  message(STATUS "Checking version of existing conan.cmake file")
+  file(READ "${CMAKE_BINARY_DIR}/conan.cmake" FILE_CONTENTS)
+  string(FIND "${FILE_CONTENTS}" "# version: ${CONAN_CMAKE_VERSION}" MATCH_RESULT)
```

Comment:
If you check the hash you should be fine?

Reply:
Good call!

Turns out the file download command checks the hash of an existing file before downloading.  If the file already exists, it returns the message "returning early; file already exists with expected SHA256 hash".  I've updated the example.